page_objects/Base_Page.py

```diff
@@ -702,18 +702,18 @@ def add_tesults_case(self, name, desc, suite, result_flag, msg='', files=[], par
             for key, value in custom.items():
                 caseObj[key] = str(value)
             Tesults.add_test_case(caseObj)
-    
+
     def make_gif(self):
         "Create a gif of all the screenshots within the screenshots directory"
         self.gif_file_name = Gif_Maker.make_gif(self.screenshot_dir,name=self.calling_module)
 
         return self.gif_file_name
-        
+
 
     def wait(self,wait_seconds=5,locator=None):
         "Performs wait for time provided"
         if locator is not None:
-            self.smart_wait(wait_seconds,locator)
+            self.smart_wait(locator,wait_seconds=5)
```

Comment:
@rahul-qxf2 - I just realized that wait should pass whatever it receives as wait_seconds to smart_wait. So this should be wait_seconds=wait_seconds ... otherwise you are ignoring the argument sent to wait().

Reply:
@qxf2 , I have corrected this. Let me know in case you have any more comments.